src/Symfony/Component/HttpFoundation/Tests/Session/Storage/Handler/MongoDbSessionHandlerTest.php

```diff
@@ -11,39 +11,46 @@
 
 namespace Symfony\Component\HttpFoundation\Tests\Session\Storage\Handler;
 
+use MongoDB\BSON\Binary;
+use MongoDB\BSON\UTCDateTime;
 use MongoDB\Client;
-use PHPUnit\Framework\MockObject\MockObject;
-use PHPUnit\Framework\SkippedTestSuiteError;
+use MongoDB\Driver\BulkWrite;
+use MongoDB\Driver\Command;
+use MongoDB\Driver\Exception\ConnectionException;
+use MongoDB\Driver\Manager;
+use MongoDB\Driver\Query;
 use PHPUnit\Framework\TestCase;
+use Symfony\Component\Clock\MockClock;
 use Symfony\Component\HttpFoundation\Session\Storage\Handler\MongoDbSessionHandler;
 
+require_once __DIR__.'/stubs/mongodb.php';
+
 /**
  * @author Markus Bachmann <markus.bachmann@bachi.biz>
  *
  * @group time-sensitive
```

Comment:
as you use `new DatetimeImmutable('now')` in the code rather than `time()`, I don't think you are compatible with the ClockMock of the phpunit bridge, which would make this group useless AFAICT.

Reply:
Removed.